workers/docker-worker/test/images/ci/VERSION

```diff
@@ -1 +0,0 @@
-0.4.4
```

Comment:
How is this specified now?

Reply:
It's similar to the other node-and-xxx docker images: `taskcluster/node-and-docker:node12.16.3`. Which is a little janky since it includes a copy of docker as well, but we can add that version to the tag when we change it -- I think there's more work to do there on checking docker-worker's docker compatibility.

Reply:
Oh, that jankiness will be addressed with the decision task and automatically building images.